xarray/tests/test_variable.py

```diff
@@ -2137,6 +2142,10 @@ class CustomIndexable(CustomArray, indexing.ExplicitlyIndexed):
         orig = Variable(dims=("x"), data=array, attrs={"foo": "bar"})
         assert isinstance(orig._data, CustomIndexable)
 
+        array = CustomWithValuesAttr(np.arange(3))
+        orig = Variable(dims=("x"), data=[array], attrs={"foo": "bar"})
```

Comment:
This needs to be `[array]` for the test to work. Is this right?

Reply:
Thanks, @dcherian, for the change! And sorry that I did not continue on this one; totally forgot about it.

Reply:
No worries! It seemed pretty close so I took a look.